doc/man7/OSSL_PROVIDER-default.pod

```diff
@@ -7,8 +7,18 @@ OSSL_PROVIDER-default - OpenSSL default provider
 =head1 DESCRIPTION
 
 The OpenSSL default provider supplies the majority of OpenSSL's diverse
-algorithm implementations.  It also acts as a fallback when no other
-provider has been loaded.
+algorithm implementations. If an application doesn't specify anything else
+explicitly (e.g. in the application or via config), then this is the
+provider that will be used as fallback: It is loaded automatically the
+first time that an algorithm is fetched from a provider or a function
+acting on providers is called and no other provider has been loaded yet.
+If another provider has already been loaded then the default provider
+won't be loaded automatically. Therefore if it is to be used in
+conjunction with other providers then it must be loaded explicitly.
+Automatic loading of the default provider only occurs once; if the default
+provider is explicitly unloaded or the explicit provider load function
+L<OSSL_PROVIDER_load(3)> is called successfully or unsuccessfully,
+the default provider will not be automatically loaded again.
```

Comment:
These are slightly separate points: i.e. automatic loading occurs only once, and it won't happen if a load attempt has already been made. The automatic loading is a maximum of once. If a load attempt has been made then it won't happen at all. How about this slight reformulation:

> If an attempt to load a provider has already been made (whether successful or not)  then the default provider won't be loaded automatically. Therefore if it is to be used in conjunction with other providers then it must be loaded explicitly. Automatic loading of the default provider only occurs a maximum of once; if the default provider is explicitly unloaded then the default provider will not be automatically loaded again.

Reply:
Much better. Thanks! Sorry for the iterations.